server/grpc.proto

```diff
@@ -25,4 +25,11 @@ service Client {
 
     rpc Receive(ReceiveRequest) returns (ReceiveResponse){
     }
+
+    rpc UpdatePartyInfo(PartyInfo) returns (PartyInfo) {
+        option (google.api.http) = {
+        post: "/partyinfo"
```

Comment:
These should be indented further IMHO due to the curly brace on the previous line